code/modules/mapping/map_template.dm

```diff
@@ -8,6 +8,15 @@
 	var/keep_cached_map = FALSE
 	var/station_id = null // used to override the root id when generating
 
+	///if true, turfs loaded from this template are placed on top of the turfs already there, defaults to TRUE
+	var/should_place_on_top = TRUE
+
+	///if true, creates a list of all atoms created by this template loading, defaults to FALSE
+	var/returns_created_atoms = FALSE
+
+	///the list of atoms created by this template being loaded, only populated if returns_created_atoms is TRUE
+	var/list/created_atoms = list()
```

Comment:
When is this cleaned, and what happens if the contents of this list get qdeleted?